### ISSUE TITLE
Not on NPM

### ISSUE THREAD

**Issue by PieterFabry** [compute-io/chisq-test#1]
This module cant be installed via npm

**Comment by kgryte** [comment]
@PieterFabry Not yet, as we still need to review and possibly make changes. In the meantime, if you need to use right now, feel free to [install](https://docs.npmjs.com/cli/install) using the Github url. If you do, be sure to use a specific commit to prevent the API from unknowingly changing.

**Comment by PieterFabry** [comment]
@kgryte When do you guys expect to put it on npm?

**Comment by chriszrc** [comment]
Any update on this?

**Comment by kgryte** [comment]
@chriszrc You may also be interested in https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/chi2gof where almost all development now occurs.

**Comment by chriszrc** [comment]
@kgryte Hi, yes we are in fact also using that library as well, but we're looking for a chi-squared test of independence (two way chi squared) which I am not finding in the stdlib package? Or am I missing something?

**Comment by kgryte** [comment]
@chriszrc Ah. Let me ping one of the stdlib core devs to see if we can get that added.

**Comment by chriszrc** [comment]
@kgryte That would be amazing!

**Comment by zross** [comment]
The chi square test at stdlib doesn't have the option of doing a test for independence right? I'm only seeing the goodness of fit.

**Comment by kgryte** [comment]
@zross That is correct. @Planeshifter is planning on implementing the test for independence this weekend. Will ping here once completed.

**Comment by kgryte** [comment]
@chriszrc @zross Chi-square independence test is now available in `stdlib`: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/chi2test

Thanks to @Planeshifter !

This is available on GitHub. Once we publish a new version to `npm`, we'll circle back again. In the meantime, feel free to try it out by installing from GitHub directly.

**Comment by chriszrc** [comment]
@kgryte @Planeshifter amazing! Going to go try it right now

**Comment by zross** [comment]
So great!

**Comment by chriszrc** [comment]
@kgryte @Planeshifter looks good! At least so far, the results are matching the output from R, thanks again!

**Comment by kgryte** [comment]
@chriszrc We test against R, which you can find [here](https://github.com/stdlib-js/stdlib/blob/f73f4e5898b578dc027ae554b404df2178a35417/lib/node_modules/%40stdlib/stats/chi2test/test/test.js#L152). If anything comes up, feel free to file an issue on the `stdlib` repository.
